_examples/golang-basics/example.gen.go

```diff
@@ -797,6 +732,86 @@ func RespondWithError(w http.ResponseWriter, err error) {
 	w.Write(respBody)
 }
 
+type method struct {
+	name        string
+	service     string
+	annotations map[string]string
+}
+
+func (m *method) Name() string                   { return m.name }
+func (m *method) Service() string                { return m.service }
+func (m *method) Annotations() map[string]string { return m.annotations }
+
+var methods = map[string]*method{
```

Comment:
- Server only (removed from client-generated code)
- Updated to pointer receiver
- Removed copying of the elements